indico/modules/designer/placeholders.py

```diff
@@ -170,7 +170,7 @@ class CategoryTitlePlaceholder(DesignerPlaceholder):
 
     @classmethod
     def render(cls, event):
-        return event.category.title
+        return event.category.title if not event.is_unlisted else _('Unlisted event')
```

Comment:
Just wondering... wouldn't an empty string be better in this case? I don't think you'd ever want "Unlisted event" printed on a badge/poster

Reply:
Makes sense!